### PR TITLE
[test][enhance] support execute multiple sql in sql file

### PR DESCRIPTION
**Pull request by 924060929**
# Proposed changes

regression testing framework support execute multiple sql in sql file

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (No)
4. Does it need to update dependencies: (Yes)
5. Are there any changes that cannot be rolled back: (Yes)

@HappenLee @dataroaring @EmmyMiao87

### CODE REVIEW

**Review by dataroaring — DISMISSED** [review]

File: regression-test/framework/pom.xml [hunk 1]
```diff
@@ -238,5 +254,10 @@ under the License.
             <artifactId>janino</artifactId>
             <version>3.1.6</version>
         </dependency>
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr4-runtime</artifactId>
+            <version>${antlr.version}</version>
+        </dependency>
     </dependencies>
 </project>
```
Comment (jackwener) [inline regression-test/framework/pom.xml:263]: ditto

File: regression-test/suites/demo/test_multiple_sqls.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+select 100;
+-- second tag is test_multiple_sqls_2
+select 200;
+-- third tag is test_multiple_sqls_3
+select 300;
+
+/* empty sql will be skipped */
+;
+;
+;
```
[the remark this answers is not in the document]
Comment (924060929) [reply regression-test/suites/demo/test_multiple_sqls.sql]: done

File: regression-test/framework/pom.xml (hunk 1 above)
Comment (924060929) [reply regression-test/framework/pom.xml]: done

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**